test/cypress/integration/manage-team-members/edit_permissions_spec.js

```diff
@@ -0,0 +1,123 @@
+'use strict'
+
+const SERVICE_EXTERNAL_ID = 'service_abc_123'
+const AUTHENTICATED_USER_ID = 'authenticated-user-id'
+const EDITING_USER_ID = 'user-we-are-editing-id'
+
+const getUserStubOpts = function (userExternalId, email, roleName, roleDescription) {
+  return {
+    external_id: userExternalId,
+    username: email,
+    email: email,
+    service_roles: [
+      {
+        service: {
+          external_id: SERVICE_EXTERNAL_ID
+        },
+        role: {
+          name: roleName,
+          description: roleDescription
+        }
+      }
+    ]
+  }
+}
+
+describe('Edit service user permissions', () => {
+  beforeEach(() => {
+    // keep the same session for entire describe block
+    Cypress.Cookies.preserveOnce('session')
+    Cypress.Cookies.preserveOnce('gateway_account')
+
+    const authenticatedUserStubOpts = getUserStubOpts(AUTHENTICATED_USER_ID, 'logged-in-user@example.com', 'admin', 'Administrator')
+    const userWeAreEditingStubOpts = getUserStubOpts(EDITING_USER_ID, 'other-user@example.com', 'admin', 'Administrator')
+    const viewOnlyUserStubOpts = getUserStubOpts('view-only-user-id', 'view-only-user@example.com', 'view-only', 'View Only')
+    const viewAndRefundUserStubOpts = getUserStubOpts('view-and-refund-user-id', 'view-and-refund-user@example.com', 'view-and-refund', 'View And Refund')
```

Comment:
do we really need `viewOnlyUserStubOpts` and `viewAndRefundUserStubOpts` for this test? we're just testing we can change the permissions for the user that relates to `viewOnlyUserStubOpts`

Reply:
this was to test the team members page, as thought we may as well, but as i said I could split this out if you don't think it makes sense as part of this spec